backend/audit/logXML.py

```diff
@@ -85,7 +85,7 @@ def logSystemEvent(infoDict):
     if 'filename' in infoDict:
         ElementTree.SubElement(systemEvent, 'filename').text = infoDict['filename']
     if 'amount' in infoDict:
-        ElementTree.SubElement(systemEvent, 'funds').text = infoDict['amount']
+        ElementTree.SubElement(systemEvent, 'funds').text = '%.2f' % infoDict['amount']
 
     prettyPrintLog(root)
```

Comment:
prettyPrintLog statement can be scrubed here, and probably removed in other functions aswell?

Reply:
I agree, I think we can just remove all the instances of prettyPrint except where it actually needs to be called

Reply:
I agree, I think we can just remove all the instances of prettyPrint except where it actually needs to be called

Reply:
Yup, I'll remove it!